packages/core/source/rule.ts

```diff
@@ -137,30 +138,64 @@ export default function parseRule(
 	return parse(rawRule.nom, context) as ReferenceNode
 }
 
-registerEvaluationFunction('rule', function evaluate(node) {
-	const firstNullableParent = node.explanation.parents.find(
-		(ref) => this.ruleUnits.get(ref)?.isNullable
+export function disablingParent(
```

Comment:
```suggestion
export function evaluateDisablingParent(
```

Pour rester dans l'idée qu'on est dans de l'évaluation.